README.md

```diff
@@ -0,0 +1,65 @@
+
+# Project Name
+
+> this project helps you keep track of reserved missions and rockets on ypor profile add or remove the on your profile.
+
+
+## Built With
+
+- react
+- bootstraps
+- redux
+
+**To get this project locally**
+
+## Run the following command on your terminal
+
+
+```
+git clone https://github.com/businesskaniki/spacex.git
+
+```
+```
+cd spacex
+
+```
+```
+npm install
+
+```
+```
+npm start
+
+```
+
+## Authors
+
+👤 **Author1**
+
+-- GitHub: [@githubhandle](https://github.com/businesskaniki)
+- Twitter: [@twitterhandle](https://twitter.com/kaniki7346)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/buinesskaniki)
+
+👤 **Author2**
+
+GitHub: [@githubhandle](https://github.com/Hatimdev)
+- Twitter: [@twitterhandle](https://twitter.com/Hatim)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/hatimdev/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] OPTIONAL
MIT license link is broken, kindly endeavor to fix it, to make your ReadMe appear more professional.